price_parser/parser.py

```diff
@@ -26,7 +26,8 @@ def amount_float(self) -> Optional[float]:
     @classmethod
     def fromstring(cls, price: Optional[str],
                    currency_hint: Optional[str] = None,
-                   decimal_separator: Optional[str] = None) -> 'Price':
+                   decimal_separator: Optional[str] = None,
+                   thousand_with_dot: bool = True) -> 'Price':
```

Comment:
Why not go with `digit_group_separator: Optional[str] = None` ([name choice reasoning](https://en.wikipedia.org/wiki/Decimal_separator#Digit_grouping)), like with the decimal separator?

It may give us some flexibility going forward.